tutorials/scripting/gdextension/gdextension_cpp_example.rst

```diff
@@ -711,6 +718,22 @@ Every second, we output our position to the console.
 Next steps
 ----------
 
-We hope the above example showed you the basics. You can
-build upon this example to create full-fledged scripts to control nodes in Godot
-using C++.
+We hope the above example showed you the basics. You can build upon this
+example to create full-fledged scripts to control nodes in Godot using C++.
+
+If you want to include other C++ libraries, consider using submodules for
+getting them into your project, in the same way as we did with ``godot-cpp``.
+Then you will need to add them to the ``SConstruct`` file. For details, consult
+the `SCons documentation <https://scons.org/documentation.html>`__, but it may
+require as little as adding these lines to the file:
+
+.. code:: python
+
+    env.Append(CPPPATH=["library-path/src/", "library-path/include/"])
+    sources = sources + Glob("library-path/src/*.cpp")
+
+If your editor is using `clangd Language Server <https://clangd.llvm.org/>`__
+to provide smart C++ features, but it doesn't know where to find the headers,
+you can fix that by `generating compile_commands.json with Scons <https://scons.org/doc/latest/HTML/scons-user/ch27.html>`__.
+This should enable actions like "go to definition", also for ``godot-cpp``
+functions and types.
```

Comment:
Adding C++ libraries is (usually) more complicated than that. I feel like this section simplifies the process too much. This section can apply to very simple libraries such as one-header libraries but not for more complex dependencies with their own build-system set up.

Reply:
Well, I'm using the typical source distribution where it's a bunch of `.h`, `.cpp` and some flags, and I've had good experience. If you're writing a dedicated piece of content, I'm happy to remove this part.

On the other hand, setting up the editor was a complete 180 to my development experience, and I think it's valuable to keep it. But I don't insist on the location.

Reply:
I think that we could point the reader to
https://docs.godotengine.org/en/latest/contributing/development/configuring_an_ide/index.html